packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -10,95 +6,117 @@ The Databricks extension for VS Code allows you to develop for the Databricks La
 
 # Features
 
+-   Create [Databricks Asset Bundle](https://docs.databricks.com/en/dev-tools/bundles/index.html)
```

Comment:
For this list, again, maybe we should match the docs more closely? It's pretty close, but: 

- Define, deploy, and run Databricks Asset Bundles to apply CI/CD patterns to your Databricks jobs, Delta Live Tables pipelines, and MLOps Stacks.
- Run local Python code files on Databricks clusters.
- Run notebooks and local Python code files as Databricks jobs. 
- Set up and configure your debugging environment and Databricks Connect using a simple checklist that triggers selection dialogs.
- Debug notebooks cell by cell with Databricks Connect.
- Synchronize local code with code in your Databricks workspace.